index.html

```diff
@@ -69,33 +67,85 @@
     SHOULD join the mailing lists below and take part in the discussions.</p>
   </section>
   <section>
+      <h2>
+        Dependencies
+      </h2>
+      <p>
+        The terms <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/semantics.html#the-link-element">link</a></dfn>, 
+        <dfn><a href="https://html.spec.whatwg.org/multipage/infrastructure.html#insert-an-element-into-a-document">inserted into a document</a></dfn>, 
+        <dfn><a href="https://html.spec.whatwg.org/multipage/infrastructure.html#resolve-a-url">resolve</a></dfn>, <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/infrastructure.html#url">url</a></dfn>, <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/semantics.html#attr-link-crossorigin">crossorigin</a></dfn>,
+        <dfn><a href="https://html.spec.whatwg.org/multipage/browsers.html#origin-2">origin</a></dfn>, <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/parsing.html#delay-the-load-event">delay the load event</a></dfn>, <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/webappapis.html#queue-a-task">queue a task</a></dfn> and <dfn><a href=
+        "https://html.spec.whatwg.org/multipage/semantics.html#concept-link-obtain">obtain the resource</a></dfn>  are defined in [[!HTML5]].
+      </p>
+      <p>
+        The terms <dfn><a href=
+        "https://www.w3.org/TR/html52/links.html#external-resource-link">external resource link</a></dfn>,<dfn><a href=
+        "https://www.w3.org/TR/html52/infrastructure.html#valid-media-query-list">valid media query list</a></dfn> and <dfn><a href=
+        "https://www.w3.org/TR/html52/infrastructure.html#match-the-environment">match the environment</a></dfn> 
+        are defined in [[!HTML52]].
+      </p>
+      <p>
+        The terms <dfn><a href=
+        "https://dom.spec.whatwg.org/#in-a-document-tree">in a document tree</a></dfn>,
+        <dfn><a href="https://dom.spec.whatwg.org/#concept-event-fire">fire an event</a></dfn> and
+        <dfn><a href=
+        "https://dom.spec.whatwg.org/#concept-node-document">node document</a></dfn>
+        are defined in [[!DOM]].
+      </p>
+      <p>
+        The term 
+        <dfn><a href=
+        "https://www.w3.org/TR/dom41/#concept-document-content-type">Content-Type metadata</a></dfn> is defined in [[!DOM41]].
```

Comment:
The main reason I changed those 4 links to DOM4.1 and HTML5.2 was because the original ones were broken... but never mind, we can fix that later when the spec gets stable.